modules/internal/pristmatic_shield_generator.json

```diff
@@ -1,12 +1,12 @@
-    { "id": "p2", "eddbID": 1489, "grp": "psg", "class": 5, "rating": "A", "cost": 7655930,   "mass": 40,  "power": 5.46, "minmass": 203, "optmass": 405,  "maxmass": 1013, "minmul": 2, "optmul": 1.5, "maxmul": 1, "recover": 1.875 },
-    { "id": "p1", "eddbID": 1490, "grp": "psg", "class": 6, "rating": "A", "cost": 24269300,  "mass": 80,  "power": 6.51, "minmass": 270, "optmass": 540,  "maxmass": 1350, "minmul": 2, "optmul": 1.5, "maxmul": 1, "recover": 2.4 },
-    { "id": "p0", "eddbID": 1491, "grp": "psg", "class": 7, "rating": "A", "cost": 76933670,  "mass": 160, "power": 7.35, "minmass": 530, "optmass": 1060, "maxmass": 2650, "minmul": 2, "optmul": 1.5, "maxmul": 1, "recover": 3 },
-    { "id": "p7", "eddbID": 1492, "grp": "psg", "class": 8, "rating": "A", "cost": 243879730, "mass": 320, "power": 8.4,  "minmass": 900, "optmass": 1800, "maxmass": 4500, "minmul": 2, "optmul": 1.5, "maxmul": 1, "recover": 3.6 }
+    { "id": "p6", "edID": 128671323, "eddbID": 1485, "grp": "psg", "class": 1, "rating": "A", "cost": 132200,    "mass": 2.5, "power": 2.52, "minmass": 13,  "optmass": 25,   "maxmass": 63,   "minmul": 2, "optmul": 1.5, "maxmul": 1, "recover": 1.2 },
+    { "id": "p5", "edID": 128671324, "eddbID": 1486, "grp": "psg", "class": 2, "rating": "A", "cost": 240340,    "mass": 5,   "power": 3.15, "minmass": 23,  "optmass": 55,   "maxmass": 138,  "minmul": 2, "optmul": 1.5, "maxmul": 1, "recover": 1.2 },
+    { "id": "p4", "edID": 128671325, "eddbID": 1487, "grp": "psg", "class": 3, "rating": "A", "cost": 761870,    "mass": 10,  "power": 3.78, "minmass": 83,  "optmass": 165,  "maxmass": 413,  "minmul": 2, "optmul": 1.5, "maxmul": 1, "recover": 1.2 },
+    { "id": "p3", "edID": 128671326, "eddbID": 1488, "grp": "psg", "class": 4, "rating": "A", "cost": 2415120,   "mass": 20,  "power": 4.62, "minmass": 143, "optmass": 285,  "maxmass": 713,  "minmul": 2, "optmul": 1.5, "maxmul": 1, "recover": 1.425 },
+    { "id": "p2", "edID": 128671327, "eddbID": 1489, "grp": "psg", "class": 5, "rating": "A", "cost": 7655930,   "mass": 40,  "power": 5.46, "minmass": 203, "optmass": 405,  "maxmass": 1013, "minmul": 2, "optmul": 1.5, "maxmul": 1, "recover": 1.875 },
+    { "id": "p1", "edID": 128671328, "eddbID": 1490, "grp": "psg", "class": 6, "rating": "A", "cost": 24269300,  "mass": 80,  "power": 6.51, "minmass": 270, "optmass": 540,  "maxmass": 1350, "minmul": 2, "optmul": 1.5, "maxmul": 1, "recover": 2.4 },
+    { "id": "p0", "edID": 128671329, "eddbID": 1491, "grp": "psg", "class": 7, "rating": "A", "cost": 76933670,  "mass": 160, "power": 7.35, "minmass": 530, "optmass": 1060, "maxmass": 2650, "minmul": 2, "optmul": 1.5, "maxmul": 1, "recover": 3 },
+    { "id": "p7", "edID": 202115326, "eddbID": 1492, "grp": "psg", "class": 8, "rating": "A", "cost": 243879730, "mass": 320, "power": 8.4,  "minmass": 900, "optmass": 1800, "maxmass": 4500, "minmul": 2, "optmul": 1.5, "maxmul": 1, "recover": 3.6 }
```

Comment:
Are you sure about this value? 202115326 is out of sequence - I would have expected 128671330.